OpenGLUnitTests/Matrix4Tests.cs

```diff
@@ -23,7 +25,7 @@ public void MatrixCreateIdentity()
         [TestMethod]
         public void MatrixCreateFromArray()
         {
-            Matrix4 matrix = new Matrix4(new double[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 });
+            Matrix4 matrix = new Matrix4(new double[] {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15});
```

Comment:
It looks like your code formatter has slightly different settings than my default.  To reduce the size of the diff and also to maintain existing code formatting could you please revert any of these white space changes?